dotnet/targets/Microsoft.Sdk.Versions.template.props

```diff
@@ -18,10 +18,10 @@
 			Include="Microsoft.@PLATFORM@"
 			TargetFramework="@DOTNET_TFM@"
 			RuntimeFrameworkName="Microsoft.@PLATFORM@"
-			DefaultRuntimeFrameworkVersion="@NUGET_VERSION_NO_METADATA@"
-			LatestRuntimeFrameworkVersion="@NUGET_VERSION_NO_METADATA@"
+			DefaultRuntimeFrameworkVersion="**FromWorkload**"
```

Comment:
I don't think this is what we want, because we end up with the same problem in .NET 8: the .NET 8 workload will point to .NET 8 packs, and we'll need to update `KnownFrameworkReference` again like you're doing below for the .NET 8 workload to load .NET 7 ref/runtime packs from the .NET 7 sdk packs.

OTOH this is for RC1 only, we can do something else in RC2+.

Using the exact versions makes everything work without having to update any KnownFrameworkReference.